sn_faucet/src/faucet_server.rs

```diff
@@ -151,7 +164,7 @@ fn deposit(root_dir: &Path) -> Result<()> {
     Ok(())
 }
 
-fn load_maid_snapshot() -> Result<Vec<MaidBalance>> {
+fn load_maid_snapshot() -> Result<HashMap<String, u32>> {
```

Comment:
better make this `HashMap<String, u32>` as a type, and have a comment explain what that `String` and `u32` is for

Reply:
Changed it to `HashMap<MaidAddress, NanoTokens>`

Is it better to also make a `type Snapshot = HashMap<MaidAddress, NanoTokens>` or is that overkill?

Reply:
Seems the type is to be used in many places.
So, I'd vote for to make it `type Snapshot = HashMap<MaidAddress, NanoTokens>` .

Reply:
Added the `Snapshot` type in https://github.com/maidsafe/safe_network/pull/1210/commits/0031538577d9c0e58e033c84d345fc8d6d0b56b6 thanks for the feedback :)